src/components/toolbar/button.js

```diff
@@ -1,15 +1,26 @@
 import React, { Component } from 'react'
 import Icon from '@mdi/react'
 import clsx from 'clsx'
+import Tooltip from '../tooltip'
```

Comment:
its better use the alias `@/components/tooltip`

Reply:
Fixed it 🙂